assets/app/view/hex.rb

```diff
@@ -54,7 +55,7 @@ def render
       props = {
         attrs: {
           transform: transform,
-          fill: Lib::Hex::COLOR.fetch(@tile&.color, 'white'),
+          fill: @user&.dig(:settings, @tile&.color) || (Lib::Hex::COLOR[@tile&.color || 'white']),
```

Comment:
this needs to be handled in borders too, try to find a clean way to do that

Reply:
"in borders"?

Reply:
Define "clean". My changed part of the code should work there, too. Is there already a map with borders?

Reply:
not sure...